src/netmanager-app/components/NetManagerMap/page.tsx

```
@@ -0,0 +1,357 @@
+        const token = process.env.NEXT_PUBLIC_MAP_API_TOKEN
+        const [isOpen, setIsOpen] = useState(false);
+        const [loading,setLoading] = useState(true)
+        const [airdata,setAirdata] = useState(false)
+        const [NodeType, setNodeType] = useState('Emoji');
+        const [mapStyle, setMapStyle] = useState(mapStyles[0].url);
+        const [countryData, setCountryData] = useState<any[]>([]);
+        const [selectedCountry, setSelectedCountry] = useState(null);
+        const [siteDetails, setSiteDetails] = useState<any[]>([]);
+        const [sessionToken, setSessionToken] = useState<string | null>(null);
+        const activeNetwork = useAppSelector((state) => state.user.activeNetwork);
+        const {MapData, Error, Loading} = useMapData()
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)